Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM docker.io/library/rust:alpine AS builder
+# RUN wget -O source.tar.gz $(wget -qO- https://api.github.com/repos/mbrubeck/agate/releases/latest | sed -nE 's/^.*"tarball_url"\s*:\s*"([^"]+)".*$/\1/p')&& tar xzf source.tar.gz && \
+#     mv mbrubeck-agate-* /agate
+
+WORKDIR /agate
+COPY src src
+COPY Cargo.toml .
+COPY Cargo.lock .
+COPY Cross.toml .
```

Comment:
Is the `Cross.toml` useful here, as we're only targeting a build for `x86_64-unknown-linux-musl`?

```suggestion
```

(As an aside I've explored using `docker buildx` to make a multi-arch container here, but I'm not familiar enough with Rust to make that work correctly. I'd be happy to explore it in a subsequent PR, if that's interesting.)